src/cdp/CdpConnection.ts

```diff
@@ -45,14 +44,14 @@ export class CdpConnection implements ICdpConnection {
   static readonly LOGGER_PREFIX_RECV = `${LogType.cdp}:RECV ◂` as const;
   static readonly LOGGER_PREFIX_SEND = `${LogType.cdp}:SEND ▸` as const;
 
+  readonly #mainBrowserCdpClient: ICdpClient;
   readonly #transport: ITransport;
 
-  /** The CdpClient object attached to the root browser session. */
-  readonly #browserCdpClient: CdpClient;
-
   /** Map from session ID to CdpClient. */
-  readonly #sessionCdpClients = new Map<Protocol.Target.SessionID, CdpClient>();
-
+  readonly #sessionCdpClients = new Map<
```

Comment:
Can we document here what `undefined` means?

Reply:
Added comment